src/c_wrapper.jl

```diff
@@ -282,27 +352,19 @@ function _unsafe_scs_solve(model::_ScsDataWrapper{S,T}) where {S,T}
         model.m,
         model.n,
         pointer_from_objref(model.A),
+        pointer_from_objref(model.P),
         pointer(model.b),
         pointer(model.c),
-        pointer_from_objref(model.settings),
     )
-    scs_set_default_settings(model.linear_solver, scs_data)
     for (key, value) in model.options
-        setfield!(
-            model.settings,
-            key,
-            convert(fieldtype(ScsSettings{T}, key), value),
-        )
+        setproperty!(model.settings, key, value)
```

Comment:
Ah. Does this do the conversion as necessary?

Reply:
it does ;)